src/coldcard.test.js

```diff
@@ -0,0 +1,149 @@
+import {ColdcardMultisigWalletConfig, WALLET_CONFIG_VERSION} from './coldcard';
+
+describe("ColdcardMultisigWalletConfig", () => {
+  const jsonConfigUUID = {
+    name: "Test",
+    addressType: "P2SH",
+    network: "testnet",
+    quorum: {
+      requiredSigners: 2,
+      totalSigners: 3
+    },
+    startingAddressIndex: 5,
+    extendedPublicKeys: [
+      {
+        name: "unchained",
```

Comment:
this might've been causing the issues with the spread operator since it's not a deep copy and these are objects within objects.